cms-xcache/Dockerfile

```diff
@@ -4,10 +4,13 @@ LABEL maintainer OSG Software <help@opensciencegrid.org>
 RUN yum install -y cms-xcache --enablerepo=osg-testing && \
```

Comment:
```suggestion
RUN yum install -y --enablerepo=osg-testing \
                cms-xcache \
                xcache-consistency-check && \
```

Reply:
For some reason It doesn't allow me to commit this change anymore, I guess because the file has changed with the other change suggestion in this same file. Anyways I've pushed a commit with the intended change